images/gh-gl-sync/SpackCIBridge.py

```diff
@@ -559,16 +597,20 @@ def sync(self):
             tags = self.list_github_tags()
 
             # Get refspecs for open PRs and protected branches.
-            open_refspecs, fetch_refspecs = self.get_open_refspecs(open_prs)
+            open_refspecs = self.get_open_refspecs(open_prs)
+            fetch_refspecs = []
             self.update_refspecs_for_protected_branches(protected_branches, open_refspecs, fetch_refspecs)
```

Comment:
At first I didn't get why we create an empty `fetch_refspecs` here, and pass it into the method below.  Why leave support for `fetch_refspecs` in `update_refspecs_for_protected_branches` but remove it elsewhere?  But now I see that method adds refspecs to the list, associated with protected branches, so we fetch them below in `fetch_github_branches`.  :+1:

Reply:
Yeah, I'm not in love with how these functions/variables have evolved, but it seems like you managed to correctly figure out the current state of things.